index.html

```diff
@@ -0,0 +1,10 @@
+<html>
+  <head>
+    <tittle>
+      <h1>This is the Gallery slide</h1>
```

Comment:
h1 tags needs to be in body not in title. Please correct it.

Reply:
yeah I will correct it